exasol/python_extension_common/deployment/language_container_deployer.py

```diff
@@ -203,8 +208,17 @@ def run(
                             For this to work either of the two conditions should be met.
                             The pyexasol connection should have an open schema, or
                             The calling user should have a permission to create schema.
+        print_alter_session_activation - If True and alter_system is False,
+                                         it will print the ALTER SESSION command to stdout.
         """
 
+        if alter_system and print_alter_session_activation:
```

Comment:
I am not 100% happy with the naming of the flag `print_alter_session_activation`.

* The printed message also describes how to alter _system_, not only _session_
* I am somehow missing either the term "SQL" or "statement". 

How about naming the flag `print_activation_statements`?

Reply:
On general (though out-of-scope of this PR), maybe enums would have been better.

E.g. 
```python 
class ActivationLevel(Enum): 
   SESSION = "session"
   SYSTEM = "system"
```

This would also avoid or simply sentences like "If True and alter_system is False, ...".

But of course, this would also be a breaking change.

Reply:
In case of ALTER SYSTEM, it does not print the SQL activation. See line 242

Reply:
Fixed